Pis-moove-it/reciclando-mobile
# Deploy mobile to beta

QA list:
- Test the app normally to see if nothing is broken

Maybe you need to run this command on the Android directory if you have an error about fastlane:
- sudo gem install fastlane -NV
